az-digital/az_quickstart
# Closes #1763, Closes #1918 - Adding config for optionally importing archived paragraphs.

## Description
When migrating from sites with workbench_moderation enabled on pages with paragraphs attached, and the latest revision of a page is a draft, and all paragraphs are marked as archived, the archived paragraphs do not migrate and are not attached to the migrated page.

## Related issues
Closes #1763
Closes #1918

## How to test
On a UA Quickstart/Quickstart 1 source site ensure workbench_moderation is enabled and there is set up for uaqs_flexible_page.
On a Arizona Quickstart/Quickstart 2 destination site
Set the configuration `drush cset az_migration.settings allow_archived_paragraphs true`
On existing migrations: lando drush migrate:import --tag="Quickstart Paragraphs" --update
On new migrations: lando drush migrate:import --group="az_migration" 
Once you have imported all revisions, you should be able to rerun the flexible page migration.
lando drush migrate:import az_node_flexible_page --update


## Types of changes

### Arizona Quickstart (install profile, custom modules, custom theme)
- Patch release changes
- Minor release changes

### Drupal core
- Patch release changes
- Minor release changes

### Drupal contrib projects
- Patch release changes
- Minor release changes

## Checklist
